package.json

```diff
@@ -83,6 +84,7 @@
     "semver": "6.3.0",
     "streamsaver": "^2.0.4",
     "style-loader": "0.23.1",
+    "swr": "^0.5.6",
```

Comment:
I wasnt familiar with `swr` but based on a quick read of the documentation, I cannot figure out why we would need it for this usecase. could you probably explain it here (or better still, in code comments)